src/Web/Telegram/API/Bot/API.hs

```diff
@@ -68,182 +73,179 @@ newtype Token = Token Text
 telegramBaseUrl :: BaseUrl
 telegramBaseUrl = BaseUrl Https "api.telegram.org" 443 ""
 
--- | Type for token
-type TelegramToken = Capture ":token" Token
-
 -- | Telegram Bot API
 type TelegramBotAPI =
```

Comment:
It is simplifying API but at the same time conceptually token is the part of API. `TelegramBotAPI` type is exposed it will be major and breaking change and can be used for telegram server mocks